content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,76 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+In the case of the Simple Them Plugin, users or theme maintainers may need to update CSS and JavaScript to match the new layout structure.
+
+==== Removed Agent Protocols
+
+Remoting changelogs for https://github.com/jenkinsci/remoting/releases/tag/remoting-4.2[4.2],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0.1[4.0.1],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0[4.0],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.40[3.40],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.39[3.39].
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.38[3.38], and
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.37[3.37]
+
+The Remoting library has been updated to resolve a security issue in remoting versions prior to 4.0.1 and to add support for the <<websocket-protocol,WebSocket protocol>>.
+The update removes outdated agent protocols that had previously been deprecated.
+`Inbound TCP Agent Protocols` versions 1, 2, and 3 have been removed.
+
+<<websocket-protocol,WebSocket protocol>> connections require agents using Remoting 4.2 or later.
+
+plugin:ssh-slaves[SSH Build Agents] will always use the latest `agent.jar`, but other agent types may need to have their `agent.jar` updated manually.
+The following agent types should be updated:
+
+* Inbound TCP Agent Protocol agents (formerly known as Java Web Start agents) with `agent.jar` / `slave.jar` on the disk should update to version 4.2 or newer.
+  The latest `agent.jar` can be downloaded from any Jenkins server at `/jnlpJars/agent.jar` as in \https://your-jenkins-server.example.com/jnlpJars/agent.jar
+* plugin:swarm[Swarm Client] should update to version 3.18 or newer
```

Comment:
Version is not correct. Any version with JNLP4 support will do the job

Reply:
@jeffret-b can you provide alternate phrasing here?  

I confirmed with the pom file that swarm client 3.18 is the first swarm client version to include remoting 4.2.  The prior version included remoting 3.40.  

Are the security changes after 3.40 only relevant for the server side?